google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineDeploymentRunConfigurationEditor.java

```diff
@@ -250,6 +250,12 @@ private void validateConfiguration() throws ConfigurationException {
           GctBundle.message("appengine.config.deployment.source.error"));
     } else if(versionOverrideCheckBox.isSelected() && StringUtils.isBlank(versionIdField.getText())) {
       throw new ConfigurationException(GctBundle.message("appengine.config.version.error"));
+    } else if (getConfigType() == ConfigType.CUSTOM) {
+      if (StringUtils.isBlank(appYamlPathField.getText()) ||
```

Comment:
We can probably separate out these error messages to show one for a missing Dockerfile and one for a missing app.yaml.